includes/User.php

```diff
@@ -2167,6 +2167,8 @@ public function setPassword( $str ) {
 				}
 				throw new PasswordError( wfMsgExt( $message, array( 'parsemag' ), $params ) );
 			}
+	            	//Delete all access tokens
+	            	$this->removeAllTokens();
```

Comment:
I think that we should limit dependencies between Helios in this file. Since Helios is mostly configured by using hooks we could add new one here and make Helios remove all tokens when this hook fires.

@macbre what is your suggestion? Should we call Helios directly or use hooks instead?